seqr/utils/elasticsearch/es_search.py

```diff
@@ -690,6 +700,11 @@ def _parse_hit(self, raw_hit):
             result['_sort'] = [_parse_es_sort(sort, self._sort[i]) for i, sort in enumerate(raw_hit.meta.sort)]
 
         self._parse_genome_versions(result, index_name, hit)
+        self._parse_xstop(result)
+
+        # If an SV has genotype-specific coordinates that differ from the main coordinates, use those
+        if is_sv and genotypes:
+            self._set_sv_genotype_coords(genotypes, result)
```

Comment:
I moved this to after `_parse_xstop` so we set the variant's end before tying to update it for specific genotypes. I did not see any translocations or complex variants with genotype-specific start/end positions but figured this was the better way to go and it resolved a bunch of test issues

Reply:
For genome SV, no genotype-specific start/end fields according to the VCF header.

Reply:
okay, but for exome there is and I didn't change the logic I just moved it. Changing/skipping this step for genomes is out of scope